pyfr/writers/vtk.py

```diff
@@ -310,6 +363,14 @@ def _write_data(self, vtuf, mk, sk):
         svpts = self._get_std_ele(name, nspts)
         nsvpts = len(svpts)
 
+        if name != 'pyr' and self.ho_output:
+            # Transform PyFR to VTK9 points
+            # See nodemaps.py for more information on
+            # why VTK9 maps are chosen over those of VTK8
+            # High order `pyr` elements are not currently
+            # supported in VTK
+            svpts = np.array(svpts)[VTKHONodeMaps.from_pyfr[name, nsvpts]]
```

Comment:
Just so my understanding is correct this is going in the opposite direction (PyFR => VTK) than what we do in the reader (Gmsh => PyFR) but both are called `from_pyfr`.

Reply:
Changed it to `to_pyfr`. Let me know if its ok now